src/linuxX64Main/kotlin/mu/Appender.kt

```diff
@@ -0,0 +1,9 @@
+package mu
+
+interface Appender {
```

Comment:
I am ok with duplication (of js code) if we don't have alternative, but isn't it possible with multiplatform to have a module which is shared by js and native? it looks like it will save a lot of copy/paste here.

Reply:
Presumably you are referring to [MPP Hierarchical Project Structure](https://blog.jetbrains.com/kotlin/2020/06/kotlin-1-4-m2-released/#hierarchical-project-structure), which isn't available in Kotlin 1.3. @oshai - Doing code sharing in the **common** module could be an option, **if** you are fine with some changes being made to the **jvm** module to match the changes made to the **common** module.

Reply:
The **Appender** interface is now defined in the **common** module.

Reply:
since as you say the hierarchial project structure is only from 1.4, I suggest to leave all duplications, and we might fix it later instead of moving files to the common module. I don't want to clutter the jvm module.

Reply:
You can currently share a source set between JS and Native. The only caveat would be that the IDE would only do intelli-sense for pure Kotlin, which is acceptable in this case.

Reply:
Reverted changes to Appender interface.